src/main/java/core/basesyntax/strategy/OperationHandlerStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface OperationHandlerStrategy {
+    public OperationHandler getOperationService(String operation);

```

Comment:
Use specific object for operation instead of `String`
```suggestion
    OperationHandler getOperationService(FruitTransaction.Operation type);

```
remove public modifier from the interface.